src/test/java/io/jenkins/plugins/google/analyze/code/security/accessor/IACValidationServiceTest.java

```diff
@@ -227,9 +229,9 @@ public void validateIAC_validParams_returnsNonEmptyViolations() throws IOExcepti
         when(closeableHttpClient.execute(eq(httpGet), any(ResponseHandler.class)))
                 .thenReturn(DUMMY_JSON_NON_EMPTY_VIOLATIONS_RES);
 
-        final List<Violation> violations = iacValidationService.validateIAC(getValidIACValidationRequest(listener));
+        final IaCValidationReport report = iacValidationService.validateIAC(getValidIACValidationRequest(listener));
 
-        assertEquals(violations, response.getResult().getValidationReport().getViolations());
+        assertEquals(report, response.getResult().getValidationReport());
```

Comment:
Have we implemented the equals method for the IACValidationReport? If not then it will just compare the object references.

Reply:
This is a generic method available in org.junit.Assert package. According to the [official documentation](https://junit.org/junit4/javadoc/4.8/org/junit/Assert.html#assertEquals(java.lang.Object,%20java.lang.Object)), it _Asserts that two objects are equal_ .